migrations/default/1_create_tables/up.sql

```diff
@@ -0,0 +1,157 @@
+
+alter table db.unit_types add constraint unit_type_owner_check
+check ((id < 90000 and owner = 'cris') or (id >= 90000 and owner = 'vz'));
+
+-------------------------------------------
+-------- Crash tables ---------------------
+-------------------------------------------
+create table db.crashes_cris (
+    crash_id integer primary key,
+    road_type_id integer references db.road_types (id)
+);
+
+-- this prevents cris from using a value that falls within
+-- our custom lookup value namespace
+alter table db.crashes_cris add constraint cris_road_type_chk
+check (road_type_id < 90000);
+
+create table db.crashes_vz (
+    crash_id integer primary key not null references db.crashes_cris (
+        crash_id
+    ) on delete cascade on update restrict,
+    road_type_id integer references db.road_types (id)
+);
+
+create table db.crashes (
+    crash_id integer unique not null references db.crashes_cris (
+        crash_id
+    ) on delete cascade on update restrict,
+    road_type_id integer references db.road_types (id)
+);
+
+
+-------------------------------------------
+-------- Unit tables ----------------------
+-------------------------------------------
+create table db.units_cris (
+    id serial primary key,
+    crash_id integer not null references db.crashes_cris (
+        crash_id
+    ) on delete cascade on update restrict,
+    unit_nbr integer not null,
+    unit_type_id integer references db.unit_types (id),
+    constraint unique_units_cris unique (crash_id, unit_nbr)
+);
+
+alter table db.units_cris add constraint cris_unit_type_chk
+check (unit_type_id < 90000);
+
+
+create table db.units_vz (
+    id integer primary key references db.units_cris (
+        id
+    ) on delete cascade on update restrict,
+    crash_id integer references db.crashes_cris (
+        crash_id
+    ) on delete cascade on update restrict,
+    unit_nbr integer,
+    unit_type_id integer references db.unit_types (id),
+    constraint unique_units_vz unique (crash_id, unit_nbr)
+);
+
+create table db.units (
+    id integer primary key references db.units_cris (id)
+    on delete cascade on update restrict,
+    crash_id integer not null references db.crashes (
+        crash_id
+    ) on delete cascade on update restrict,
+    unit_nbr integer not null,
+    unit_type_id integer references db.unit_types (id),
+    constraint unique_units unique (crash_id, unit_nbr)
+);
+
+-------------------------------------------
+-------- People tables --------------------
+-------------------------------------------
+create table db.people_cris (
+    id serial primary key,
+    unit_id integer references db.units_cris (
+        id
+    ) on delete cascade on update restrict,
+    crash_id integer references db.crashes_cris (
+        crash_id
+    ) on delete cascade on update restrict,
+    prsn_nbr integer not null,
+    unit_nbr integer not null,
+    is_primary boolean default false,
+    constraint unique_people_cris unique (unit_id, prsn_nbr)
+);
```

Comment:
we need the `crash_id` column to exist on the `people_cris` table in order to join up the unit + crash data properly. 
this will allow us to run the cris import without querying our DB to pull down unit IDs before inserting people records.

so, if we don't like having the `crash_id` on the `people_cris` table (but not the vz or unified people tables), the alternative is to remove it completely and deal with this in the import ETL.